modules/tinymce/src/themes/silver/test/ts/browser/sidebar/SidebarTest.ts

```diff
@@ -14,7 +15,7 @@ interface EventLog {
 
 describe('browser.tinymce.themes.silver.sidebar.SidebarTest', () => {
   const store = TestStore<EventLog>();
-  const hook = TinyHooks.bddSetupLight<Editor>({
+  const createEditor = async () => await McEditor.pFromSettings<Editor>({
```

Comment:
If adding and removing editors is important, the design intent of `TinyHooks` is you would have multiple `context` blocks inside the `describe` block and a new `hook` for each one. `bddSetupLight` uses `before` and `after` to create and remove the editor like you're doing manually here.

This also avoids having `it` blocks as siblings of `context` blocks, which will probably cause weird state bugs at some point.

Reply:
Thanks for the knowledge. I've so far only worked with McEditor when initializing editors with different configs for testing, so I appreciate this alternative.

Reply:
I'm glad you worked it out! I'm sure this stuff is documented somewhere, we should make sure it's in the onboarding process.